src/parse-jobs.js

```diff
@@ -3,106 +3,98 @@ const cheerio = require('cheerio');
-            if (!updatedItem) {
-                log.error(`Not found job listing id ${request.uniqueKey} in search results`);
-                return;
-            }
+    if (!clearDetails) {
+        // so for now second option will be used as jobDetails, decoding below was for json.description
+        clearDetails = jsonCompanyInfo.initialState.jlData.job.description; // html encoded, decoding is not 99% accurate
+        try {
+            clearDetails = entities.decode(clearDetails); // this will transform html decoded content to plain html
+            clearDetails = $(clearDetails).text(); // then we create html from content and getting it as plain text
+        } catch (err) {
+            log.error(err);
+        }
+    }
 
-            // remove html formatting from job description
-            // for whatever reason div below sometimes available, sometimes not
-            let clearDetails = $('#JobDescriptionContainer').text().trim(); // but no artifacts from html decoding here
-            if (!clearDetails) {
-                // so for now second option will be used as jobDetails, decoding below was for json.description
-                clearDetails = json.description; // html encoded, decoding is not 99% accurate
-                try {
-                    clearDetails = entities.decode(clearDetails); // this will transform html decoded content to plain html
-                    clearDetails = $(clearDetails).text(); // then we create html from content and getting it as plain text
-                } catch (err) {
-                    log.error(err);
-                }
-            }
-            // get employer id, check if we have cached overview
-            const eid = $('#EmpHero').data('employer-id');
-            let moreDetails = companyDetails[eid];
-            if (!moreDetails) {
-                moreDetails = {};
-                // get company details from company overview page
-                const companyUrl = new URL($('div.logo.cell a').attr('href'), BASE_URL);
-                log.info(`company overview GET ${companyUrl}`);
-                const rq2 = await Apify.utils.requestAsBrowser({
-                    url: companyUrl,
-                    proxyUrl,
-                    ...REQUEST_HEADERS,
-                });
-                $ = cheerio.load(rq2.body);
-                $('div.infoEntity', '#EmpBasicInfo').each((i, el) => {
-                    const infoKey = $('label', el).text().trim();
-                    const info = $('span', el).text().trim();
-                    if (infoKey && info) {
-                        moreDetails[infoKey] = info;
-                    }
-                });
-                companyDetails[eid] = moreDetails;
-            }
+    // ON A HUGE NUMBER OF SEARCH ITEMS JSON IS A BIT DIFFERENT. SO NEEDS TO MAKE ALL THIS CHECKS
+    const moreDetails = {
+        logo: jsonCompanyInfo.initialState.jlData.header.employer.squareLogoUrl
```

Comment:
everything here can be written using nullish coalescing and optional chaining, it's more readable as well:
```js
logo: jsonCompanyInfo?.initialState?.jlData?.header?.employer?.squareLogoUrl ?? null,
```
then remove the ternary operator